osutil/group.go

```diff
@@ -156,3 +184,28 @@ func FindGid(group string) (uint64, error) {
 	//return strconv.ParseUint(group.Gid, 10, 64)
 	return strconv.ParseUint(group, 10, 64)
 }
+
+// FindGid returns the identifier of the given UNIX group name.
+func FindGroup(gid uint64) (string, error) {
+	group, err := lookupGroupByGid(gid)
+	if err != nil {
+		return "", err
+	}
+	return group, nil
+}
+
+// FindGroupOwning obtains UNIX group ID and name owning file `path`.
+func FindGroupOwning(path string) (uint64, string, error) {
```

Comment:
And this as well? Again, something simple is enough for me. E.g. create a file, ensure its the group  of the user running the test and call this helper or something like this?